infra/app/env-template/main.tf

```diff
@@ -205,6 +213,10 @@ module "staff" {
       name  = "LOWDEFY_SECRET_OPENID_DOMAIN",
       value = "https://cognito-idp.${data.aws_region.current.name}.amazonaws.com/${module.staff_idp.user_pool_id}/.well-known/openid-configuration",
     },
+    {
+      name  = "MATOMO_URL_BASE",
+      value = var.analytics_url,
+    }
```

Comment:
Are you instrumenting the staff portal? I don't think we have any code on the staff portal side to support this.

Reply:
Ah, good catch. we don't have any code in the staff portal to sup